packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -70,21 +116,21 @@ There are three URIs that you need to consider when creating an OIDC app for the
 
 1. **Login Redirect URI** &mdash; after being successfully authorized by Okta, this is the callback location where the user is directed to along with the authorization code. This URI must exactly match the Redirect URI value pre-registered in the Okta app integration settings
 2. Optional. **Initiate Login URI** &mdash; this URI is used if the application is launched from the Okta dashboard (known as an IdP-initiated flow) and you want your Okta integration to handle redirecting your users to your application to start the sign-in request. When end users click your app in their Okta dashboard, they are redirected to the `initiate_login_uri` of the client application, which constructs the authentication request and redirects the end user back to the authorization server. This URI must exactly match the Initiate URI value pre-registered in the Okta app integration settings
-3. Optional. **Post Logout Redirect URI** &mdash; this URI is a location to send the user after a sign off operation is performed and their session is terminated; otherwise the user is redirected back to the Okta sign-in page
+3. Optional. **Post Logout Redirect URI** &mdash; this URI is a location to send the user after a sign off operation is performed and their session is terminated; otherwise, the user is redirected back to the Okta sign-in page
```

Comment:
```suggestion
3. Optional. **Post Logout Redirect URI** &mdash; a location to send the user after a sign off operation is performed and their session is terminated. Otherwise, the user is redirected back to the Okta Sign-In Page.
```

Reply:
Removed "Okta" from the sign-in page (as it could be a non-Okta sign-in page), so leaving it as lower case. Other suggested changes made.